internal/execute/watcher.go

```diff
@@ -54,7 +54,7 @@ func (w *watcher) hasErrorsInTsConfig() bool {
 			}
 			return true
 		}
-		if !reflect.DeepEqual(w.options.CompilerOptions(), configParseResult.CompilerOptions()) {
+		if !reflect.DeepEqual(w.options.CompilerOptions().SourceFileAffecting(), configParseResult.CompilerOptions().SourceFileAffecting()) {
```

Comment:
```suggestion
		if !reflect.DeepEqual(w.options.CompilerOptions().Clone(), configParseResult.CompilerOptions().Clone()) {
```

Actually, this should be good enough for a hotfix.

Reply:
From what I could see, the issue was that one of the two `CompilerOptions` objects being compared had `<nil>` for one of its last fields (which I think is the config cache?), whereas the other one has the cache defined.

I don't see how `.Clone()` could fix this, unless `Clone` doesn't actually do what it says?